cw721-timelock/src/testing/tests.rs

```diff
@@ -244,6 +228,34 @@ fn test_locked_nft() {
     assert_eq!(claim_res, ContractError::LockedNFT {});
 }
 
+#[test]
+fn test_claim_non_existent_timelocked_nft() {
+    let mut deps = mock_dependencies_custom(&[]);
+    let msg = InstantiateMsg {
+        kernel_address: MOCK_KERNEL_CONTRACT.to_string(),
+        owner: Some("creator".to_owned()),
+        authorized_token_addresses: None,
+    };
+    let info = mock_info(MOCK_CW721_CONTRACT, &[]);
+    let env = mock_env();
+
+    instantiate(deps.as_mut(), env.clone(), info.clone(), msg).unwrap();
+
+    // Attempt to claim a non-existent time-locked NFT
+    let claim_msg = ExecuteMsg::ClaimNft {
+        cw721_contract: AndrAddr::from_string(MOCK_CW721_CONTRACT.to_string()),
+        token_id: "non_existent_token".to_string(),
+    };
+
+    let claim_res = execute(deps.as_mut(), env.clone(), info.clone(), claim_msg);
```

Comment:
I'd suggest using `.unwrap_err()` then asserting equality with the contract error type

Reply:
I solved this.